develop.sh

```diff
@@ -47,6 +53,15 @@ elif [ "$1" == "npm" ]; then
     npm_install
     exit
 
+elif [ "$1" == "spark" ]; then shift
+    if [ "$1" == "build" ]; then
```

Comment:
if you have no containers, then I'm not sure if this is going to run as expected. You're going to do `docker-compose run`, and docker-compose will see that the container doesn't exist, and build it. then it'll run it. then it'll execute your command a few lines down, and run `docker-compose build` again, building the same containers that you just built.
Additionally, this will format the namenode _every time_ you do a build. Is that something that you want? It might make sense to put this in a separate `init` method that you can call just once, or when you need to wipe your setup and start again

Reply:
separate init method which will be called just one in shell script? I am not sure if I follow this :(

Reply:
I don't think this is still an issue, there's a different spark format command for now, we'll look into automating that in the future.